core/drivers/fabric-driver/makefile

```diff
@@ -22,21 +22,22 @@ protos-js:
 		cp -r ../../../common/protos-js/relay ./protos-js/ && \
 		cp -r ../../../common/protos-js/package.json ./protos-js/
 
-weaver-fabric-interop-sdk:
+weaver-fabric-interop-sdk: protos-js
 	mkdir -p weaver-fabric-interop-sdk && \
 		cp -r ../../../sdks/fabric/interoperation-node-sdk/build ./weaver-fabric-interop-sdk/ && \
 		cp -r ../../../sdks/fabric/interoperation-node-sdk/index.js ./weaver-fabric-interop-sdk/ && \
+		cp -r ./protos-js ./weaver-fabric-interop-sdk/ && \
```

Comment:
Would using `ln -s` be better? 
I'm still not sure why do we need this at all, if we copy `package.json` instead of `package-local.json` I think it would be fine...
Because `sdk` is built using `package.json` even in local-build and that works fine. Only `npm install` requires `package-local.json`.
Can you re-try this? copying `package.json` in line 30, removing line 29, and build? Usually dependencies' dependencies are resolved by common node_modules folder, the sdk doesn't need to copy its own dependencies, if has driver has it

Reply:
The `make build-local` command to build a local (non-docker) executable was failing without these changes. It complained about not finding a `package.json` in `weaver-fabric-interop-sdk/protos-js/`, because that subfolder didn't even exist.  I think this wasn't covered by the tests.